### ISSUE TITLE
Zoned into a concrete wall

### ISSUE THREAD

**Issue by butterrmahjely** [NCC-Lykos/neocron-classic#215]
This was a minor bug, I zoned from P3 to P1 and found this gigantic wall blocking me. I ended up re-zoning and it was fine.

![shot0006](https://user-images.githubusercontent.com/6628124/36995707-ccb16ed6-2082-11e8-8333-c220cbf73c1d.png)

**Comment by NCC-Lykos** [comment]
Looks like you were whipped around facing the wrong direction

**Comment by butterrmahjely** [comment]
yeah, again it was a minor bug, I think this can be closed if you feel the need to.
